README.md

```diff
@@ -208,14 +208,18 @@ new GsonBuilder()
 
 ## Download
 
-Add a Gradle dependency to the `apt` and `provided` configuration.
+Add a Gradle dependency to the `annotationProcessor`/`kapt` and `implementation`/`api` configuration.
 
 ```kotlin
-annotationProcessor("com.ryanharter.auto.value:auto-value-gson:1.2.0")
+annotationProcessor("com.ryanharter.auto.value:auto-value-gson-extension:1.2.0")
 implementation("com.ryanharter.auto.value:auto-value-gson-runtime:1.2.0")
 
 // Optional @GsonTypeAdapterFactory support
 annotationProcessor("com.ryanharter.auto.value:auto-value-gson-factory:1.2.0")
+
+// Legacy generic artifact that includes both -extension and -factory above. Should not be used
+// unless necessary.
```

Comment:
Do you want to add a note explaining that you give up incremental compilation if you use this?  The "don't use this" note sounds ominous.

Reply:
Good point, will add something. Incremental compilation works, it's just aggregating instead of isolating

Reply:
Oh, I thought that incremental depended on isolating processors, so this broke it.  That's good that it doesn't break incremental compilation, though.

Reply:
2bcf03e